### PR TITLE
Start implementing drawImage

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Hopefully mostly a case of lifting code from OpenRCT2.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
I'm going to stop this PR here and do the next bit in a new PR as then it minimises the amount that can go wrong.

File: src/OpenLoco/Graphics/Gfx.h [hunk 1]
```diff
@@ -88,6 +88,16 @@ namespace OpenLoco::Gfx
     };
 
 #pragma pack(pop)
+    namespace G1ElementFlags
+    {
+        constexpr uint16_t hasTransparancy = 1 << 0; // Image data contains transparent sections (when not set data is plain bmp)
+        constexpr uint16_t unk1 = 1 << 1;            // Unknown function not used on any entry
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Graphics/Gfx.h:94]: Is this true for RCT2 as well as Locomotion?

**Review by AaronVanGeffen — APPROVED** [review]
Took this for a quick test and haven't noticed any issues. No noticeable impact on frame rates either. Tree wilting is fine, too.

Could you amend the commit message when merging?

File: src/OpenLoco/Graphics/Gfx.h (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.h:94]: Yeah I've never found a sprite that has this flag set in either rct2 or locomotion

**Pull request merged**